server/index.js

```diff
@@ -1,39 +1,53 @@
+require('newrelic');
 const path = require('path');
 const parser = require('body-parser');
 const express = require('express');
 const cors = require('cors');
-const { Client } = require('pg');
+const { Pool } = require('pg');
 
-const client = new Client({
+console.log('NODE_ENV:', process.env.NODE_ENV);
```

Comment:
You should remove the console log if this is the final iteration of the index.js script.